src/app/checkout/checkout-context.tsx

```diff
@@ -30,10 +39,16 @@ function checkoutReducer(state: State, action: Action) {
       return { ...state, numberOfTickets: action.payload };
     }
 
-    /** @todo add actions for manipulating payment info */
+    case 'updatePayment': {
+      console.log('acton.payload', action.payload);
```

Comment:
For `select` values, this currently comes in as
```js
{ expirationDate.year: '2025' }
```
And the state does not get updated.